dashboard/db/migrate/20170923000355_populate_storage_id.rb

```diff
@@ -0,0 +1,24 @@
+class PopulateStorageId < ActiveRecord::Migration[5.0]
+  def change
+    # We don't want to run as part of our deploy if the table is large (i.e. prod)
+    # Prod will instead do this using the very similar oneoff script
+    # bin/oneoff/backfill_data/channel_tokens_storage_id
+    return if ChannelToken.count > 100000
```

Comment:
Not sure if this is the best way to accomplish this. Could instead check if env != production.

Asher's approach for an equivalent change seemed to be to get devs to run this manually https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/db/migrate/20170818182341_required_storage_app_id_in_channel_tokens.rb

Reply:
I prefer the check to env instead.

Reply:
+1 for checking env, this should happen automatically to developers unless there's a reason that doesn't work.

Reply:
I'm surprised not to find any existing migrations where we check environment, though I think it should be as simple as `Rails.env.production?`.